PointOfInteraction/SmartSpot/doc/spec.md

```diff
@@ -0,0 +1,70 @@
+# SmartSpot
+
+## Description
+
+Smart Spots are devices which provide the technology to allow users to interact (suggestions mailbox, co-creation, …) or to obtain extra information (tourism, infotainment, …). The data model contains resources to configure the interaction service such as the broadcasted URL (probably will be a shortened url), the interval between broadcasts (frequency), the availability of the service, transmission power in function of the area to cover, etc.
+
+In addition to the presented data model, this entity inherits the Device data model. This means that by hierarchy, the SmartSpot entity is a child or specialization of a Device.
```

Comment:
The SmartSpot entity type is a subtype of `Device` and as a result it can be the subject of any of the properties that an entity of type `Device` may have.